src/main/scala/pipelines/images/cifar/RandomCifar.scala

```diff
@@ -33,17 +33,18 @@ object RandomCifar extends Serializable with Logging {
     // Set up a filter Array.
     val filters = DenseMatrix.rand(conf.numFilters, conf.patchSize*conf.patchSize*numChannels, Rand.gaussian)
 
-    val featurizer =
+    val unscaledFeaturizer =
       new Convolver(filters, imageSize, imageSize, numChannels, None, true)
-        .then(SymmetricRectifier(alpha=conf.alpha))
-        .then(new Pooler(conf.poolStride, conf.poolSize, identity, _.sum))
-        .then(ImageVectorizer)
-        .then(new Cacher[DenseVector[Double]])
-        .thenEstimator(new StandardScaler).fit(trainImages)
-        .then(new Cacher[DenseVector[Double]])
-
-    val labelExtractor = LabelExtractor then
-      ClassLabelIndicatorsFromIntLabels(numClasses) then
+        .andThen(SymmetricRectifier(alpha=conf.alpha))
```

Comment:
can we switch to "andThen" at the end of the line - I greatly prefer this style.